.gitignore

```diff
@@ -0,0 +1,19 @@
+.venv
+.vscode
+build
+ 
+# documentation artifacts
+_build/
+_images/
+_static/
+_templates/
+_toc.yml
+docBin/
+_doxygen/
+_readthedocs/
+ 
+# avoid duplicating contributing.md due to conf.py
+docs/CHANGELOG.md
```

Comment:
why are these lines needed? These could be legit files that we want in the repository.